util/fhir.ts

```diff
@@ -39,12 +38,26 @@ export function createPatientResourceString(birthDate: string): string {
   return JSON.stringify(pt, null, 2);
 }
 
-export function getFhirResourceString(resource: fhir4.Resource) {
-  const primaryCodePath = parsedPrimaryCodePaths[resource.resourceType].primaryCodePath;
-  return `(${fhirpath.evaluate(resource, `${primaryCodePath}.coding.code`)}: ${fhirpath.evaluate(
-    resource,
-    `${primaryCodePath}.coding.display`
-  )})`;
+/**
+ * Identifies the primary code path of a resource and constructs a string which displays
+ * resource summary information depending on what is a available
+ * @param resource {Object} a fhir DataRequirement object
```

Comment:
I think we might want to edit this doc string to say "a fhir Resource object" - I believe we are passing in a created resource rather than the data elements